package.json

```diff
@@ -39,11 +39,13 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/dom": "^7.30.0",
     "eslint": "^7.21.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0"
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "typescript": "^4.2.3"
```

Comment:
So, we will be using TypeScript in this project?

Reply:
yarn peer dependency